rootfs/opt/kubectl/example/GoogleBackup/manifest/statefulset-backup.tpl.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: <TIER>
+  labels:
+    tier: <TIER>
+    role: backup
+    app: qserv
+spec:
+  selector:
+    matchLabels:
+      tier: qserv
+      role: backup
+      app: qserv
+  serviceName: qserv
+  podManagementPolicy: "Parallel"
+  replicas: <WORKER_COUNT>
+  template:
+    metadata:
+      labels:
+        tier: qserv
+        role: backup
+        app: qserv
+    spec:
+      containers:
+        - name: sdk
+          image: "google/cloud-sdk:238.0.0"
+          imagePullPolicy: Always
+          command:
+            - <CMD>
```

Comment:
This is somewhat obtuse, since you're using a generic image, and the script isn't here either.  It seems like this either gets to be '/config-backup/backup.sh' or '/config-backup/restore.sh'  You can probably put this directly in jinja and say if 'restore then make it this value' so that you can easily discover what the right thing is.

Reply:
I'll add a comment to yaml here in the short term.